kivy/lang/parser.py

```diff
@@ -422,6 +422,10 @@ def execute_directives(self):
                     ref = ref[6:].strip()
                     force_load = True
 
+                # if #:include [force] "path with quotes around"
+                if ref[0] == ref[-1] and ref[0] in ('"', "'"):
+                    ref = ref[1:-1]
```

Comment:
I did the commits as 3 alternatives i.e. to choose from. I thought it was wrong because it allows multiple quotes without handling the amount of quotes _and_ if they are closed, thus it might cause confusion as for why does it even work if someone forgets to add a quote and it'd be this:

    #:include [force] """C:\my path""